md/widgets.md

```diff
@@ -9,6 +9,36 @@ A page consists of widgets that are arranged in rows, and optionally, containers
 
 Each widget in the palette has a description. To show a widget description, just roll over the widget in the palette.
 
+## Widget Actions
+
+:::info
+ Not available for containers.
+:::
+
+Close to the widget name in the properties panel, several options are available in widgets actions button.
+
+* Use **view** menu option to view the code of standard widget. Use standard widgets as a template for a custom widget, click on **Save as**, find a name for the custom widget, and make it your own.
+
+* Use **edit** menu option, only on custom widget, to go to widget editor.
+
+* Use the **switch** menu option to replace a widget by another one and keep the current configuration.
+
+Image below shows wizard of switch widget behaviour
+
+![widget_switch_mapping](images/widget_switch_mapping.png)
+ 
+In this wizard, select or type the widget to switch to, then click **Show properties**.
+
+Whenever possible, properties in the source widgets are mapped with similar properties in the target widget.
+ 
+ You can edit that or map more, then click on **Switch**.                    
+
+When the switch is completed, chances are you will still have properties to fill out, in the properties panel.
```

Comment:
"chances are you will still have properties to fill out"
=> "you still may have to set properties from the properties panel"